src/linear_algebra/basic.lean

```diff
@@ -2102,6 +2102,22 @@ end
 
 end field
 
+/-- Given `f : M₁ ≃ₗ[R] M₂` and `U` a submodule of `M₂`, `f.comap U` is the
+induced `linear_equiv` from `U.comap f.to_linear_map` to `U`. -/
+@[simps]
+def comap [semiring R] [add_comm_monoid M] [semimodule R M]
```

Comment:
I think there should be a `section` above that already has these variables. Can you find it and move this def there?

Reply:
Done!